src/main/java/TheThinker.java

```diff
@@ -0,0 +1,74 @@
+public class TheThinker {
+
+    public static final String NAME = "TheThinker";
+
+    public static void main(String[] args) {
+
+        printGreeting();
+        String userInput;
+
+        do{
+            userInput = UserInputParser.getUserInput();
+            String userAction = UserInputParser.parseUserAction();
+            doTaskAccordingToUserAction(userAction);
+
+        }while(!userInput.equals("bye"));
```

Comment:
use consistent spacing,
eg:
in line 5 you have:
public static void main(String[] args) {
in line 18 you have:
public static void printGreeting(){